containers/be-redis-pm2/Dockerfile

```
@@ -31,34 +31,40 @@ LABEL git="https://github.com/lehuygiang28/vphim"
+# Install Redis and Supervisor
 RUN apk update && \
-  apk add --no-cache redis curl
+  apk add --no-cache supervisor redis
 
 WORKDIR /usr/src/app
 
-# Install cloudflared
-RUN curl -L --output cloudflared https://github.com/cloudflare/cloudflared/releases/latest/download/cloudflared-linux-amd64 && \
-  chmod +x cloudflared && \
-  mv cloudflared /usr/local/bin/
+RUN npm install -g pm2
```

Hadolint

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)